common/src/main/java/cz/incad/kramerius/audio/servlets/ServletAudioHttpRequestForwarder.java

```diff
@@ -120,8 +120,11 @@ private void forwardData(InputStream input, ServletOutputStream output) {
                     && (ex.getCause().getMessage().equals(CONNECTION_RESET) || ex.getCause().getMessage().equals(BROKEN_PIPE))) {
                 LOGGER.warning("Connection reset probably by client (or by repository)");
             } else {
-                LOGGER.log(Level.SEVERE, null, ex);
-            }
+                if ("ClientAbortException".equals(ex.getClass().getSimpleName())) {
```

Comment:
`getClass()` by se nemělo používat. 

Před `} catch (IOException ex) {` 
by mělo jít použít:
 ```
} catch (ClientAbortException ex) {
   LOGGER.info("Bad sound unit image");
}
```
pro zachycení této vyjímky.

Reply:
@jkremlacek To v tomto případě nejde, protože ClientAbortException je součástí codebase Tomcatu, kterou jsem nechtěl začleňovat přímo do Krameria. Bohužel :-(

Reply:
A `ex.getCause().getMessage()` obsahuje v případě ClientAbortException co? Nedalo by se toho chytit?

Reply:
Nevím, není to zdokumentované https://tomcat.apache.org/tomcat-8.0-doc/api/org/apache/catalina/connector/ClientAbortException.html

Muselo by se to vyzkoušet, trochu se bojím, že se ta message může měnit - přes verze nebo dokonce přes lokalizace.

Spring framework to řeší vnitřně velmi podobně:
https://github.com/spring-projects/spring-framework/blob/42382996612825c96c745a593716f31446075a5c/spring-websocket/src/main/java/org/springframework/web/socket/sockjs/transport/session/AbstractSockJsSession.java#L332

Reply:
Pak to tam asi budeme moct propašovat, jen bych teda k tomu dal komentář, že je to kvůli vyhnutí se referenci Cataliny - kdyby se nad tím někdo v budoucnu pozastavoval, proč to tam je.